### PR TITLE
Flexible configs per script instance

### PR DESCRIPTION
**Pull request by przadka**

### CODE REVIEW

**Comment by nesta-onur111** [comment]
cool 😎 @przadka
and i suggest you should recheck some logics for finding rebased branches, merged branched without PR etc
thanks for that

**Comment by przadka** [comment]
@nesta-onur111 why do you think the logic should be rechecked?  anything in particular that you think may not work as expected?

**Comment by nesta-onur111** [comment]
@przadka 
I haven't tested this project in a real situation, so there may be unexpected issues with the git branches that I can't identify at the moment. You should run the bot for a while to uncover and address any problems that arise

**Comment by ppolewicz** [comment]
@przadka needs rebase

**Comment by przadka** [comment]
@ppolewicz its rebased now

File: config_subtensor.yaml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+DATABASE_DIR: "db-subtensor"
+MAIN_REPO: "opentensor/subtensor"
+FORKS:
+  - "mz-Datura/subtensor"
+  - "backend-developers-ltd/subtensor"
```
Comment (ppolewicz) [inline config_subtensor.yaml:5]:
```suggestion
  - "backend-developers-ltd/subtensor-fork"
```

File: .gitignore [hunk 2]
```diff
@@ -3,4 +3,8 @@ path/to/venv/
 */__pycache__/
 */*/__pycache__/
 .env
-/venv/
\ No newline at end of file
+/venv/
+
+# main_repo.db and repo_fam.db are created by database.py to store the state of the repository
+main_repo.db
+repo_fam.db
```
Comment (ppolewicz) [inline .gitignore:10]: You ignore files here, but this PR now adds these files:

```
db-opentensor/main_repo.db
db-opentensor/repo_fam.db
```
Which I don't think should be the case.

File: config_subtensor.yaml (hunk 1 above)
Comment (przadka) [reply config_subtensor.yaml]: config removed from the repo

File: .gitignore (hunk 2 above)
Comment (przadka) [reply .gitignore:11]: removed, i added them by mistake

**Comment by nesta-onur111** [comment]
current logic for finding rebased commits that i implemented before is wrong.
and there was a little issue for finding new commits, but now it's fixed.
we should define the correct logic to find the rebased commits.

**Review by ppolewicz — APPROVED** [review]

**Pull request merged**